src/samples/doxygen/buffer.cpp

```diff
@@ -0,0 +1,52 @@
+#include <rpp/rpp.hpp>
+#include <iostream>
+
+std::ostream & operator<<(std::ostream & out, const std::vector<int>& list)
+{
+    out << "{";
+    auto size = list.size();
+    for (int i = 0; i < size; ++i)
```

Comment:
Not critical in this case, but just a tip: use size_t when you deal with size of containers

Reply:
Oh yep, I should have used `auto` for compiler to automatically match the type.

Reply:
No, in such an case it would be also int =) because of `auto i = 0` will be deduced as int because of `0` is int. It would work in case of `auto i = size`, but not in this case

Reply:
You're right!